Unix/http/httpclientauth.c

```diff
@@ -1098,7 +1137,14 @@ Http_CallbackResult HttpClient_IsAuthorized(_In_ struct _HttpClient_SR_SocketDat
 
             PAL_Free(input_token.value);
 
-            if ((maj_stat == GSS_S_CONTINUE_NEEDED) || (maj_stat == GSS_S_COMPLETE)) {
+            if (maj_stat == GSS_S_COMPLETE)
+            {
+                self->authorizing = FALSE;
+                self->isAuthorized = TRUE;
+                return PRT_CONTINUE;
+            }
+            else if (maj_stat == GSS_S_CONTINUE_NEEDED)
+            {
```

Comment:
It looks like everything else was formatting, this was the only actual executable change, right?